src/Sfx-Standalone/.build/files.msi.wxs.xslt

```diff
@@ -0,0 +1,148 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
this is crazy... we have to do all of these to get things work?

Reply:
Well, WIX not a active project anymore and SFX is a ongoing project. So there may be more requirements coming. It would be better if we follow the pattern now before the code become too big to refactor.

Reply:
What pattern? Just saying there should be a better way to do packing/installation than WIX :)

Reply:
I mean the WIX pattern. Yeah, we may need to find a better packing way on Windows.

Reply:
Just gave WIX a second look. Seems WIX is now back to active again.